src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,24 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+export const PostInfo = ({ post }) => {
```

Comment:
Since we use more than 2 fields from `post` object, let's use destructing
```suggestion
export const PostInfo = ({ post }) => {
const { title, user, ...} = post 
```